src/assets/mapStyles/waterTemperature/mapStyles.js

```diff
@@ -159,7 +66,21 @@ export default {
                 },
                 'showButtonLayerToggle': true
             },
+            {
```

Comment:
changing the background to be the water color instead (since we drop the water baselayer from openmaptiles) makes the states blue, so have to add this solid fill state layer